src/framework/shortcuts/view/editshortcutmodel.cpp

```diff
@@ -39,42 +38,45 @@ EditShortcutModel::EditShortcutModel(QObject* parent)
 
 void EditShortcutModel::load(const QVariant& originShortcut, const QVariantList& allShortcuts)
 {
-    clear();
+    TRACEFUNC;
+
+    clearNewSequence();
+
+    m_allShortcuts = allShortcuts;
+    m_potentialConflictShortcuts.clear();
 
     QVariantMap originShortcutMap = originShortcut.toMap();
-    QString originCtx = originShortcutMap.value("context", ANY_CONTEXT).toString();
-    bool isOriginCtxAny = originCtx == ANY_CONTEXT;
+    std::string originCtx = originShortcutMap["context"].toString().toStdString();
 
     for (const QVariant& shortcut : allShortcuts) {
-        if (isOriginCtxAny) {
-            m_potentialConflictShortcuts << shortcut;
+        if (shortcut == originShortcut) {
             continue;
         }
 
         QVariantMap map = shortcut.toMap();
-        QString ctx = map.value("context", ANY_CONTEXT).toString();
+        std::string ctx = map["context"].toString().toStdString();
```

Comment:
Avoid using the `[]` operator for reading, because in this case, the container detach occurs